english/visit-hagrid/visit-hagrid.md

```diff
@@ -8,4 +8,16 @@ Hermione responds by:
 
 [Yelling](../yell-at-snape/yell-at-snape.md)
 
-[Punching Snape](../punch-snape/punch-snape.md)
\ No newline at end of file
+[Punching Snape](../punch-snape/punch-snape.md)
+
+Snape fails to hide his shock at Herminoe's behaviour, and assumes she must be pre menstural, because he had heard women tend to act crazy at certains times of the month. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.

Reply:
Ah ha! It's alive!